themis/spec/secure-cell/design.md

```diff
@@ -0,0 +1,69 @@
+---
+weight: 10
+title:  Secure Cell design
+---
+
+# Secure Cell design and properties
+
+First of all, let's discuss the desired properties of the cryptosystem.
+
+  - Strong confidentiality and integrity guarantees.
+
+    Given the typical use cases for Secure Cell,
+    it must prevent direct and indirect disclosure of protected data
+    in addition to preventing inadvertent modifications to it.
```

Comment:
You realise that –– when published — these lines don't have `\n` between them but rather form a long sentence? Just saying.

Reply:
> You realise that –– when published — these lines don't have \n between them but rather form a long sentence?

Yes, it's intentional. It's called semantic line-breaking. This makes editing text easier: moving sentences and clauses around, etc.

If a hard line break is really necessary – which is almost never – there's always `<br>`.